### DISCUSSION TITLE
Bootcamp Driven

### DISCUSSION THREAD

**Discussion by G4bb1** [frontendbr/forum#2327] [Perguntas e Respostas]
Alguém sabe me dizer se o Bootcamp da Driven é bom? 

Agradeço desde já.

**Comment by felquis** [comment]
Olá, tudo bem? Qual é o seu nome?

**Reply by G4bb1** [reply]
Oi, tudo bem. Meu nome é Gabriela.

**Reply by felquis** [reply]
Não conheço este Bootcamp, de uma olhada em outros tópicos para ver se encontra algo relacionado https://github.com/frontendbr/forum/discussions?discussions_q=Bootcamp 👍
